include/tscpp/util/Convert.h

```diff
@@ -0,0 +1,46 @@
+/** @file
+
+  Collection of utility functions for converting between different chars.
```

Comment:
Why not call it CharConvert.h?

Reply:
How about `StingConvert.h` ?

Reply:
Other candidate is `TextConvert.h`, but it sounds like utils for the TextView?